ring_test.go

```diff
@@ -172,6 +172,34 @@ var _ = Describe("Redis Ring", func() {
 			Expect(ringShard2.Info().Val()).To(ContainSubstring("keys=100"))
 		})
 	})
+
+	Describe("shard passwords", func() {
+		It("can be initialized with a single password, used for all shards", func() {
+			ring = redis.NewRing(&redis.RingOptions{
+				Addrs: map[string]string {
+					"shard1": "host1",
+				},
+				Password: "some-password",
+			})
+
+			Expect(ring).ToNot(BeNil())
+		})
+
+		It("can be initialized with a passwords map, one for each shard", func() {
+			ring = redis.NewRing(&redis.RingOptions{
+				Addrs: map[string]string {
+					"shard-1": "host-1",
+					"shard-2": "host-2",
+				},
+				Passwords: map[string]string{
+					"shard-1": "pwd-1",
+					"shard-2": "pwd-2",
+				},
+			})
+
+			Expect(ring).ToNot(BeNil())
```

Comment:
Same here - this does not test anything.

Reply:
:+1: